Assets/Mirror/Editor/ReadOnlyDrawer.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+using UnityEditor;
+
+namespace Mirror
+{
+    [CustomPropertyDrawer(typeof(ReadOnlyAttribute))]
+    public class ReadOnlyDrawer : PropertyDrawer
+    {
+        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        {
+            GUI.enabled = false;
```

Comment:
We should cache the value with 
bool isEnabled = GUI.enabled
before setting it to false and then set it to
GUI.enabled = isEnabled
instead of setting it to true

Reply:
Done.